<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>480</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>WatchU</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(252, 252, 244)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>480</height>
    </size>
   </property>
   <widget class="QLabel" name="label_sn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>학      번</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_en">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>197</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>시험번호</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_en">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquare</family>
     </font>
    </property>
    <property name="accessibleName">
     <string>lineEdit_en</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(252, 252, 252)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="accessibleName">
     <string>startButton</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 230, 153)</string>
    </property>
    <property name="text">
     <string>응 시 하 기</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquare</family>
     </font>
    </property>
    <property name="accessibleName">
     <string>lineEdit_sn</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(252, 252, 252)</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
